### ISSUE TITLE
Software installation instructions

### ISSUE THREAD

**Issue by tavareshugo** [cambiotraining/metagenomics#3]
@lkalmar can you list here on this issue all the software that is used in the course?
So we can keep track of things for future iterations of the course.

**Comment by lkalmar** [comment]
R packages:
dada2, phyloseq, Biostrings, ggplot2, reshape2, readxl, tydiverse

Command line applications:
fastqc, multiqc, cutadapt, trimmomatic, bowtie2, samtools, metaphlan, mash, SPAdes, clumpify.sh (part of the bbmap package), flash (for merging reads), maxbin2, checkm (database!), gtdbtk (database!), prokka, abricate (database!)

**Comment by tavareshugo** [comment]
Can I check: 

1. It's `checkm` (version 1), not `checkm2`?
2. Do you have commands/links to download all those databases?

**Comment by lkalmar** [comment]
> Can I check:
> 
> 1. It's `checkm` (version 1), not `checkm2`?
> 2. Do you have commands/links to download all those databases?

1. it is the original checkm, that is still the gold standard, but I will look into checkm2
2. I think all of these are conda installable, but to be sure, I will collect all the installations here.

**Comment by tavareshugo** [comment]
For point 2. I just meant the databases, not the software itself. 
For example, with CheckM2 they have a command `checkm2 database --download --path <output>`.

I don't think CheckM (version 1) has a command, but there is this: https://data.ace.uq.edu.au/public/CheckM_databases/
Is that the correct database to download?

For the other programs, I don't know if the databases come with the software or if they need to be installed separately.

**Comment by tavareshugo** [comment]
`gtdbtk` database can be obtained with the `download-db.sh` command

**Comment by lkalmar** [comment]
Yes, checkm has a database and you need to set the database path. Abricate comes with the database, but there is a way to update those [here](https://github.com/tseemann/abricate?tab=readme-ov-file#updating-the-databases)

**Comment by lkalmar** [comment]
Otherwise, the update is perfect, please close the issue

**Comment by tavareshugo** [comment]
Reopening as we are missing MetaPhlan database. 

In [this page](https://github.com/biobakery/MetaPhlAn/wiki/MetaPhlAn-4#installation) they recommend for conda installations: 

```bash
metaphlan --install --bowtie2db <database folder>
```

Does this look right @lkalmar?

In the future we would then need to adjust the materials to point to a `--bowtie2db` folder that we decide to save the database into.

**Comment by lkalmar** [comment]
You either install the DB in your conda / miniconda / mamba / micromamba folder with the simple command `metaphlan --install` (not recommended on the HPC, but on own computer or in-house server this is simpler).

Or, you define the database path with the above mentioned `metaphlan --install --bowtie2db <database folder>` but in that case you have to define the database path during the run.

**Comment by tavareshugo** [comment]
during installation for Trinidad workshop:

> might be worth working out some alternate instructions for manual installation of the metaphlan indices as it's not trivial and the installation script seems like a dice roll

**Comment by tavareshugo** [comment]
it's a bad idea to have everything in the same environment, due to dependency conflicts (e.g. an old version of `maxbin`). 

Update the instructions to have each software in a separate environment.
